src/continuum_absorption/absorption_H.jl

```diff
@@ -313,63 +312,52 @@ function _H2plus_bf_and_ff(ν::Real, T::Real, nH_I_div_partition::Real, nH_II::R
-provided in [Bates (1952)](https://ui.adsabs.harvard.edu/abs/1952MNRAS.112...40B/abstract).
-
-The combined H₂⁺ bound-free and free-free linear absorption coefficient can be cast as:
-```
-α = const * σ₁ * exp(-U₁ / (kboltz*T)) * n(H I, n=1) * n(H II) * (1 - exp(-hν/(kboltz*T)))
-```
-where σ₁ and U₁ are just functions of ν. Note that Bates (1952) and Gray (2005) both use the number
-density of all energy states of H I instead of n(H I, n=1). However, Kurucz (1970) makes a note in
-section 5.2 about how the photodisociation of H₂⁺ produces a ground state H atom and that we should
-therefore use n(H I, n=1) instead. This difference will only cause Bates/Gray to be wrong by a
-fraction of a percent (at most) for T ≤ 1.2e4 K. In this function we use n(H I, n=1).
-
-In this function, we use polynomial approximations that Gray (2005) fit for σ₁ and U₁ using data
-from Table 1 of Bates (1952). Gray (2005) claims that the approximations match Bates' (1952)
-tabulated absorption data for 3800 Å ≤ λ ≤ 25000 Å at an accuracy of 0.3%. After reviewing the data
-in Bates (1952) I've concluded that Gray (2005) actually rounded down the lower end of the
-wavelength from 10⁵/26 Å (or ∼3847 Å) to 3800 Å. In other words, the approximation is good for
-3847 Å ≤ λ ≤ 25000.0 Å.
-
-While this may fit to σ₁ and U₁ to better than 0.3%, comparisons against values from table 2 of
-Bates (1952) indicate that the function only reproduces the full "absorption coeffient" at better
-than 1.5%. Gray (2005) did not specify a temperature range, but Bates (1952) only computed the
-absorption coefficient for 2500 K ≤ T ≤ 12000 K. In the text they have a comment that the proton's
-De Broglie wavelength is large at 2500 K and their semi-classical treatment may start to break
-down. However, as long as we use n(H I, n=1), there doesn't seem to be any reason for us to enforce
-an upper temperature limit.
-
-Bates (1952) states that his classical treatment of the interaction is probably most accurate at
-higher temperatures and longer wavelengths (note that the longest λ considered in the paper is 8
-times larger than the max λ that the polynomials are fit against). He suggests that treatment is
-probably correct "to well within one part in ten even at the lower temperatures and [lower
-wavelengths]."
+This computes the H₂⁺ number density from those of H I and H II, since ionized molecules 
+are not included in the molecular equlibrium calculation.  Stancil provides approximate equilibrium 
+constants, K, for the molecule, but the Barklem and Collet values used elsewhere by Korg may be more 
+reliable.  Once ionized molecules are fully supported, those values should be used instead. While 
+cross sections are tabulated down to only 3150 K, the cross sections could be linearly interpolated 
+1000 K or so lower, if reliable K values are availble.
+
+Becauce n(H₂⁺) is computed on the fly, the "cross-sections" used in this function have units of 
```

Comment:
```suggestion
Because n(H₂⁺) is computed on the fly, the "cross-sections" used (internally) by this function have units of 
```
2 minor suggestions:
- fix the typo: "Becauce" -> "Because"
- i made a slight tweak to try to make it clear that the cross-sections are only used internally (to make it clear that it's not something most users should care about). But definitely feel free to ignore this if you disagree